webpack.config.dev.js

```diff
@@ -30,24 +25,20 @@ module.exports = {
       },
       {
         test: /\.scss$/,
-        loader: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: [
-            'css-loader?modules&importLoaders=2&localIdentName=[path][local]__[hash:base64:5]',
-            'resolve-url-loader',
-            'sass-loader',
-          ],
-        }),
+        use: [
+          'style-loader',
+          `css-loader?modules&importLoaders=1&localIdentName=${css.localIdentName}`,
+          'resolve-url-loader',
+          'sass-loader',
+        ],
       },
       {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: [
-            'css-loader?importLoaders=1&localIdentName=[path][local]__[hash:base64:5]',
-            'resolve-url-loader',
-          ],
-        }),
+        use: [
+          'style-loader',
+          `css-loader?importLoaders=1&localIdentName=${css.localIdentName}`,
```

Comment:
Split this into an object with options?
```
{
  loader: 'css-loader',
  options: {
    importLoaders: 1,
    localIdentName: css.localIdentName,
  }
}
```

Reply:
This is basically how it was before, I just wanted a way to change them all at the same time easily (in webpack 3 I dont even think you can do what you say here, you need to use `LoaderOptions` plugin which is a headache)